packages/common/src/authorization/types.ts

```diff
@@ -38,10 +44,11 @@ type Subject =
     | 'DashboardComments'
     | 'CustomSql'
     | 'CompileProject'
+    | 'SemanticViewer'
```

Comment:
In the PR we are adding `SemanticViewer` permissions to be used alongside the already existing `SavedChart` permissions.

The `SemanticViewer` permissions are used to control access in to the `SemanticLayerService` while `SavedChart` is used just to control chart creation from `SemanticViewerSavedChartService`

There is quite some overlap here and not sure if we couldn't make do with just `SavedChart` permissions and just not add any `SemanticViewer` specific permissions. On the other hand, having the two could add flexibility in the future, no use case in mind yet.